CHANGES.rst

```diff
@@ -18,13 +18,23 @@ Release history
    - Deprecated
    - Removed
 
-3.4.5 (unreleased)
+3.5.0 (unreleased)
 ------------------
 
 *Compatible with Nengo 3.0 - 3.2*
 
 *Compatible with TensorFlow 2.2 - 2.9*
 
+**Added**
+
+- Added support for Python 3.9/3.10. (`#224`_)
```

Comment:
I consider these "changes" and not additions/removals; there's nothing to point to that was added or removed aside from metadata. So I would either add both of these to a `**Changed**` section or in that section add something like `Updated supported Python versions to 3.7 - 3.10.`